src/components/vue-form-renderer.vue

```diff
@@ -148,7 +148,12 @@ export default {
 
         if (JSON.stringify(this.transientData) !== JSON.stringify(this.data)) {
           this.$emit('update', this.transientData);
-          this.watchDataChanges(this.transientData);
+
+          //Verify changes in the Data for the watchers call
+          let diff = this.dataChanges(this.transientData, this.data);
```

Comment:
`diff` could be instantiated with `const` instead of `let` since it doesn't seem to be re-assigned.